Week-2-Assignment.R

```diff
@@ -1,10 +1,14 @@
 #UCB ADMISSIONS DATASET
 
+
+
 UCBAdmissions
 #1
 View(UCBAdmissions)
+
 sum(UCBAdmissions)
-summary(UCBAdmissions)
```

Comment:
Good Job, but do not delete summary(), let it be there!

Reply:
ok Sir, Thank you